storage/types.go

```diff
@@ -192,6 +192,46 @@ func NewNodeIDFromBigInt(depth int, index *big.Int, totalDepth int) NodeID {
 	}
 }
 
+func NewNodeIDFromBigInt(depth int, index *big.Int, totalDepth int) NodeID {
+	if got, want := totalDepth%8, 0; got != want || got < want {
+		panic(fmt.Sprintf("storage NewNodeFromBitInt(): totalDepth mod 8: %v, want %v", got, want))
+	}
+
+	if totalDepth == 0 {
+		panic("totalDepth must not be zero")
+	}
+
+	// Put index in the LSB bits of path.
+	// This code more-or-less pinched from nat.go in the golang math/big package:
+	_S := bits.UintSize / 8
```

Comment:
nit: Could declare this const.
```suggestion
	const _S = bits.UintSize / 8
```

Reply:
done